### PR TITLE
#162781153 Filter floors by block

### PR DESCRIPTION
**Pull request by caveinn**
**What does this PR do?**
-  Enables an admin to filter floors by block

**Description of task to be completed**
- Ensure that as an admin I can filter the floors by blocks

**How should this be tested?**
- Run the server `http://127.0.0.1:5000/mrm`.
- Run the `filterByblock` query  passing a `blockId` that is oresent in the datbase as shown below

 ```
 query {
  filterByBlock(blockId: 1){
    id
    name
    blockId
}
}
```

**Screenshots**
image

**What are the relevant pivotal tracker stories?**

[#162241777](https://www.pivotaltracker.com/story/show/162781153)

### CODE REVIEW

**Comment by RonKbS** [comment]
Nice work @caveinn.

Is the Alembic file necessary if no changes to the models that create tables in the database have been introduced?

**Comment by caveinn** [comment]
> Nice work @caveinn.
> 
> Is the Alembic file necessary if no changes to the models that create tables in the database have been introduced?

My tests were failing locally without including them but I have consulted and been informed  that I should ignore it when pushing to the remote.

File: api/floor/schema.py [hunk 1]
```diff
@@ -109,6 +110,14 @@ def resolve_get_rooms_in_a_floor(self, info, floor_id):
         rooms = query.filter(RoomModel.floor_id == floor_id)
         return rooms
 
+    @Auth.user_roles('Admin')
+    def resolve_filter_by_block(self, info, blockId):
+        query = Floor.get_query(info)
+        floors = query.filter_by(block_id=blockId)
+        if not floors:
```
Comment (Bonifase) [inline api/floor/schema.py]: Also, write a test for this case

Comment (caveinn) [reply api/floor/schema.py]: @Bonifase I have implemented the requested change.

Comment (Bonifase) [reply api/floor/schema.py]: Use Andela engineering-playbook as a guide for your PR description

Comment (caveinn) [reply api/floor/schema.py]: Sure, I will get on to that immidietly.

**Pull request merged**